pyproject.toml

```diff
@@ -85,7 +85,26 @@ all = [
     {include-group = "dev"}
 ]
 
-
+[tool.hatch.envs.default]
+features = ["all"]
```

Comment:
You don't want the default hatch environment to be minimal?

Reply:
OK, I can set the default to minimal, and include all the features in the other environments.